pkg/util/crypt.go

```diff
@@ -0,0 +1,12 @@
+// +build !linux
+
+package util
+
+import (
+	"errors"
+)
+
+func Crypt(key string, salt string) (string, error) {
+	// We use the GNU reentrant form of crypt() ...
+	return "", errors.New("crypt() password hashes are not supported on this platform")
```

Comment:
Hmmm, I don't think this works since we need to compare it from the htpasswd unit tests as well.